### DISCUSSION TITLE
uhh, why is this happening?

### DISCUSSION THREAD

**Discussion by NitinIsFit** [termux/termux-app#2800] [Q&A] [answered]
```js
~ $ terminal-onclick-url-open=true
terminal-onclick-url-open=true: command not found
~ $ terminal-margin-vertical=10dp
terminal-margin-vertical=10dp: command not found
```

**Comment by USLTD** [comment]
What's this?

**Reply by NitinIsFit** [reply]
> What's this?

This is some command which I tried to run, but it didnt work, its been so long and i dont remember what i was doing, but maybe this command was from some tutorial or official docs, or maybe it was mentioned in recent release or a little earlier than recent release, but NVM, i dont want to do this thing anymore ¯\_(ツ)_/¯

**Comment by csd4ni3l** [comment]
you cant have - in variable names
thats the only thing i can think of

**Reply by NitinIsFit** [reply]
Do i have to write those commands without using `-`?

**Reply by NitinIsFit** [reply]
If yes, then i will try it later someday ¯\_(ツ)_/¯

**Comment by agnostic-apollo** [comment]
That needs to be added in `~/.termux/termux-properties` file and is not a command.

https://github.com/termux/termux-app/releases/tag/v0.118.0

**Reply by NitinIsFit** [reply, marked as answer]
I don't understand

**Reply by NitinIsFit** [reply]
Looks like you want me to do some of my own research, well okay

**Reply by NitinIsFit** [reply]
> That needs to be added in `~/.termux/termux-properties` file and is not a command.
> 
> https://github.com/termux/termux-app/releases/tag/v0.118.0

What needs to be added in `~/.termux/termux-properties` file and is not a command?

**Reply by USLTD** [reply]
@Nit-nit You need to add `terminal-onclick-url-open=true` and other *"commands"* you were trying to run in `~/.termux/termux.properties`.
For more information please refer to [this wiki](https://wiki.termux.com/wiki/Terminal_Settings)
